contrib/ckan/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# docker-compose build && docker-compose up
+ckan:
+  container_name: ckan
+  image: ckan/ckan:latest
```

Comment:
This should be `codeforafrica/ckan:latest` - https://github.com/CodeForAfricaLabs/openAFRICA#updating-docker-image